File: baekjoonn/11651_좌표 정렬하기 2/B_11651.py
import sys


N = int(input())
arr = []
for i in range(N):
    x, y = map(int, sys.stdin.readline().split())
    arr.append([x, y])

arr.sort(key=lambda a: (a[1], a[0]))
for idx, num in enumerate(arr):
    num = map(str, num)
    sys.stdout.write(' '.join(num) + "\n")

File: baekjoonn/11651_좌표 정렬하기 2/B_11651.js
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n')

const N = parseInt(input[0])
let arr = []
for (let i = 1; i <= N; i++) {
    const temp = input[i].split(" ").map(v => Number(v))
    arr.push(temp)
}

arr.sort((a, b) => {
    if (a[1] !== b[1]) {
        return a[1] - b[1]
    } else {
        return a[0] - b[0]
    }
})

let sortedArr = ''
for (let i = 0; i < N; i++) {
    sortedArr += `${arr[i][0]} ${arr[i][1]}\n`
}
console.log(sortedArr)
